webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js

```diff
@@ -106,7 +108,29 @@ const DetailsCard = ({
                 emptyState={<DefaultLoaderEmptyState />}
                 status={status}
               >
-                {hostgroupName}
+                <a
+                  href={foremanUrl(
+                    `/hostgroups/${hostgroupId}-${(hostgroupName || '').replace(
+                      '.',
+                      '-'
+                    )}/edit`
+                  )}
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  {hostgroupName}
+                </a>
+                &nbsp;
+                <a
+                  href={foremanUrl(
+                    `/hosts?search=hostgroup="${hostgroupName}"`
+                  )}
+                  className="disabled-text"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  <i className="fa fa-search" />
```

Comment:
I guess search icon here for is not the best option but I couldn't came up with anything better,
I'm open to suggestions!

Reply:
how about `pficon-server-group` if we have that available?